aws-services/uninstall-aws-services.hbs.md

```diff
@@ -0,0 +1,104 @@
+# Uninstall AWS Services
+
+This topic tells you how to uninstall the AWS Services package from the Tanzu Application Platform (commonly
+known as TAP) package repository.
+
+## <a id="prepare-infra"></a> Step 1: Prepare your infrastructure
+
+Before uninstalling the package, you will have to decide what to do with the existing resources you have
+claimed.
+
+* In case you want to keep a resource:
+
+  1. Identify the corresponding Crossplane managed resource:
+
+      ```console
+      kubectl get instances.rds.aws.upbound.io
+      ```
+
+  1. Edit the resource:
+
+      ```console
+      kubectl edit instances.rds.aws.upbound.io <RESOURCE-NAME>
+      ```
+
+  1. Change the resource `deletionPolicy` to `Orphan`:
```

Comment:
Nitpick, perhaps could mentioned the original value of `deletionPolicy` here, and phrase the sentence like Change the resource `deletionPolicy` from `VALUE` to `Orphan`:

Reply:
I like it - done!